CHANGELOG.md

```diff
@@ -36,6 +36,10 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#4792](https://github.com/thanos-io/thanos/pull/4792) Store: Fix data race in BucketedBytes pool.
 - [#4769](https://github.com/thanos-io/thanos/pull/4769) Query-frontend+api: add "X-Request-ID" field and other fields to start call log.
 
+### Changed
+
+- [#4864](https://github.com/thanos-io/thanos/pull/4864) Remove the old PromQL editor
```

Comment:
```suggestion
- [#4864](https://github.com/thanos-io/thanos/pull/4864) UI: Remove the old PromQL editor
```

It'd be nice if we can prefix this with `UI: `. More context for people reading changelog.